python/tvm/testing.py

```diff
@@ -0,0 +1,12 @@
+""" TVM testing utilities """
+import numpy as np
+
+def assert_allclose(actual, desired, rtol=1e-7, atol=1e-7):
```

Comment:
I suggest putting this function under https://github.com/dmlc/tvm/tree/master/topi/python/topi/testing

Since this is only used within topi test.

Reply:
Agree, including tvm.testing in a patch for TOPI is misleading. But ideally, we need all tests to use non-zero `atol` by default so I still think we need testing utilities on TVM level. Please consider reviewing updated patchset, where all the components are present. tvm.testing is now in TVM patch.